python/xgrammar/kernels/apply_token_bitmask_inplace_cpu.py

```diff
@@ -1,35 +1,60 @@
 """CPU implementation for in-place applying token mask."""
 
-import time
 from typing import List, Optional, Union
 
 import torch
 
 from ..base import _core
 
 
+# Error messages
+class BitmaskValidationError(ValueError):
+    """Raised when bitmask validation fails."""
+
+    @staticmethod
+    def dims_error(tensor_name: str, dims: int) -> str:
+        return f"{tensor_name} should be 1D or 2D, but got {dims}D"
+
+
+LOGITS_CPU_ERROR = "logits must be on CPU"
+BITMASK_CPU_ERROR = "bitmask must be on CPU"
+LOGITS_TYPE_ERROR = "logits must be of type float32"
+BITMASK_TYPE_ERROR = "bitmask must be of type int32"
+MAX_DIMS = 2
```

Comment:
Some of this was just clean up from my linter to get rid of ruff warnings around const strings in errors and having specific error types.

Reply:
We can skip the TRY rules for Ruff. There are two reasons:
1. Separating error messages from where they occur could make it harder to maintain
2. We also could throw errors from C++, but the customized exception in Python cannot cover them